scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -289,10 +289,11 @@ def build_graph(df, dataset):
     df = muon_selections.veto_electrons(df)
     df = muon_selections.apply_met_filters(df)
     if args.makeMCefficiency:
-        df = df.Define("GoodTrigObjs", "wrem::goodMuonTriggerCandidate(TrigObj_id,TrigObj_filterBits)")
-        df = df.Define("passTrigger","(HLT_IsoTkMu24 || HLT_IsoMu24) && wrem::hasTriggerMatch(goodMuons_eta0,goodMuons_phi0,TrigObj_eta[GoodTrigObjs],TrigObj_phi[GoodTrigObjs])")
+        df = df.Define("GoodTrigObjs", f"wrem::goodMuonTriggerCandidate(TrigObj_id,TrigObj_filterBits, \"{era}\")")
```

Comment:
@cippy I had the initial impression that it was not needed, however, without the quotations, it's interpreted as an integer. I stumbled upon this while running the 2016 wf, since it has PostVFP in it and it cannot be interpreted as int. Otherwise for 2018 it was working. And double quotes will be needed since it will evaluated in a C++ expression.

Reply:
Ok, then it can be kept as it is, no problem